<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxControl</class>
 <widget class="QWidget" name="AxControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}

QPushButton
{
	font: 18px;
    color: black;
	background-color: rgb(113, 113, 198);
	border:1px solid white;
	border-radius: 5px;
}

QLabel
{
	font: 16px &quot;微软雅黑&quot;;
	color: white;
	border:2px solid white;
	background-color:black;
}</string>
  </property>
  <widget class="QLabel" name="CH1Namelbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH6Namelbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH1ASIgnallbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainType2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TC2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH1lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AIN1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH4Namelbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH5lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AIN5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH3Namelbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH5Namelbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainType1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TC1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH7Namelbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH8lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AIN8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH2lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AIN2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH2Namelbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH3lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AIN3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH4lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AIN4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH6lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AIN6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>通道</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH8Namelbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH7lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AIN7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>信号名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH2ASIgnallbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH3ASIgnallbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH4ASIgnallbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH5ASIgnallbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH8ASIgnallbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH7ASIgnallbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH6ASIgnallbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH1BSIgnallbl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH4BSIgnallbl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH3BSIgnallbl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH8BSIgnallbl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>240</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH6BSIgnallbl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH5BSIgnallbl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH2BSIgnallbl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CH7BSIgnallbl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
